File: Alphabetical Addition.dart
String addLetters(List<String> letters) {
  var alpha = "zabcdefghijklmnopqrstuvwxy";
  List letters_c = letters+["z"];
  var count = 0;
  for(var i =0;i<letters_c.length;i++){
    count+=alpha.indexOf(letters_c[i]);
  }
  return alpha[count%26];
}

File: Alphabetical Addition/Alphabetical Addition.swift
func addLetters(_ letters: [Character]) -> Character {
  let alpha = "zabcdefghijklmnopqrstuvwxy"
  let letters_c = letters + ["z"]
  var count = 0
  for i in 0..<letters_c.count {
      if let index = alpha.firstIndex(of: letters_c[i]) {
          count += alpha.distance(from: alpha.startIndex, to: index)
      }
  }
  let resultIndex = count % 26
  let targetIndex = alpha.index(alpha.startIndex, offsetBy: resultIndex)
  return alpha[targetIndex]
}
